src/app/home/components/DataList/Plans/MigrationsTable.tsx

```diff
@@ -77,7 +44,10 @@ export default class MigrationsTable extends React.Component<any, any> {
       isFailed: false,
       isSucceeded: false,
     };
-
+    if (plan.spec.persistentVolumes && !migration.spec.stage) {
```

Comment:
PVs can be still be copied during a stage, just not moved. This condition doesn't cover that case.